sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_mcps.c

```diff
@@ -410,15 +437,29 @@ void gnrc_lorawan_mcps_request(gnrc_lorawan_t *mac,
     int waiting_for_ack = mcps_request->type == MCPS_CONFIRMED;
 
     gnrc_lorawan_build_uplink(mac, pkt, waiting_for_ack,
-                              mcps_request->data.port);
+                              mcps_request->data.port, mcps_request->data.adr);
```

Comment:
as discussed offline some months ago, ADR is not configured on a packet basis. Therefore ADR shouldn't be a setting of the mcps_request